build_container/docker_build.sh

```diff
@@ -1,6 +1,49 @@
 #!/bin/bash
+#The ppc64le is not supportted by google-cloud-sdk. So ppc64le is temporary removed.
+IMAGE_ARCH=("amd64" "arm64")
+
+buildImage()
```

Comment:
style nit: build_image (we're following Google Bash Style)

Reply:
@lizan 
Has been modified. Please check it. Thanks.